orchagent/main.cpp

```diff
@@ -445,6 +453,10 @@ int main(int argc, char **argv)
     DBConnector config_db("CONFIG_DB", 0);
     DBConnector state_db("STATE_DB", 0);
 
+    // Instantiate ZMQ server 
```

Comment:
If ZMQ architecture is DASH-specific, should we skip creating the zmq_server for the normal device? Otherwise is there any security concern for extra exported ports?

Reply:
Yes, as my understand create ZMQ server code will only merge to dash branch.
In master branch, we don't use ZMQ.

Reply:
Will by default disable ZMQ.
Will send emaill notify this change.